### PR TITLE
Created and added an Australian 915-928MHz band handler based on LoRa Alliance 1.0.1 Edition Section 7.5

### PR DESCRIPTION
**Pull request by Mehradzie**
Hey Orne,

I went a head and created the Aussie edition of the band for you. I have tested this and it's all sweet. 
Would love it if you wish to use it in your amazing LoraServer. :D 
[Frequency UP DOWN mapping.xlsx](https://github.com/brocaar/loraserver/files/229356/Frequency.UP.DOWN.mapping.xlsx)

Cheers,

Mehrad

### CODE REVIEW

**Comment by brocaar** [comment]
Awesome! However, could you make this pull-request to https://github.com/brocaar/lorawan? The ``vendor`` folder includes external dependencies and will be overwritten on updates. After it has been merged into https://github.com/brocaar/lorawan, I'll update the ``vendor`` folder in loraserver :-)

**Comment by Mehradzie** [comment]
Good point. Didn't think of that. I'll do it in the first chance.

Mehrad

**Comment by brocaar** [comment]
I've implemented this in https://github.com/brocaar/lorawan/commit/5c2323c5760f0a4fa48847c03210f22e7c428e89

File: vendor/github.com/brocaar/lorawan/band/band_au915_928.go [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+	}
+
+	// initialize downlink channel 0 - 7
+	// In Australia this is set from 10-13 if we reside in offset 0
+	// Mentioned in
+	// LoRaWAN 1.0. Standard - Section 7.5.7
+	for i := 0; i < 8; i++ {
+		DownlinkChannelConfiguration[i] = Channel{
+			Frequency: 923300000 + (i * 600000),
+			DataRates: []int{10, 11, 12, 13},
```
Comment (brocaar) [inline vendor/github.com/brocaar/lorawan/band/band_au915_928.go]: This should be DR8-DR13 I think. The offset value will be derrived from ``RX1DROffsetConfiguration``.

Comment (Mehradzie) [inline vendor/github.com/brocaar/lorawan/band/band_au915_928.go]: Thanks

**Pull request closed without merging**